styles.css

```diff
@@ -1,590 +1,1169 @@
+    font-size: 18px;
+    font-weight: 400;
+    border: 2px solid #396df2;
+    color: #fff;
+    width: 40%;
+    border-radius: 10px 12px;
+    height: 30px;
+    padding-left: 20px;
+    padding-right: 20px;
+    padding-top: 9px;
+    margin-left: 5px;
+    margin-right: 5px;
+    background-color: #396df2;
+  }
+
+  .resume {
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    color: #396df2;
+    background: #fff;
+    height: 50px;
+    padding-left: 3%;
+    margin-left: 2%;
+  }
+
+  button {
+    width: 50%;
+    border: 1.5px  solid #396df2;
+    border-radius: 10px 10px;
+    margin-bottom: 40px;
+  }
+
+  .resume:active {
+    background-color: #fff;
+  }
+
+  .resume:hover {
+    background-color: #637fc4;
+  }
+
+  .lang {
+    padding-left: 5px;
+    padding-top: -1px;
+    color: #000;
+    font-weight: 500;
+    line-height: 24px;
+    font-style: normal;
+    font-size: 20px;
+  }
+
+  .langs {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .about-me {
+    padding-bottom: 1px;
+    background-color: #ffff;
+    width: inherit;
+  }
+
+  .abt-self {
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    padding-top: 8%;
+    text-align: left;
+    color: #091e42;
+  }
+
+  .seperate-1 {
+    padding-left: 6%;
+    padding-right: 2%;
+  }
+
+  .seperate-2 {
+    padding-top: 3.5%;
+    padding-left: 2%;
+    padding-right: 6%;
+  }
+
+  .works1 {
+    display: flex;
+    flex-direction: row;
+    padding-top: 4%;
+    padding-bottom: 6%;
+  }
+
+  .abt-p {
+    padding-top: 0;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    word-wrap: break-word;
+  }
+
+  .languages {
+    padding-left: 2px;
+    padding-right: 2px;
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+  }
+
+  .special-2 {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .languages > ul {
+    list-style: none;
+  }
+
+  .framework {
+    background-color: #ffff;
+    gap: 12%;
+  }
+
+  .frameworks {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    color: #344563;
+    border-bottom: 1.6px solid #dfe1e6;
+    line-height: 24px;
+  }
+
+  .skills {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: #000;
+    border-bottom: 1.6px solid  #dfe1e6;
+    padding-bottom: 19px;
+  }
+
+  .icon1 {
+    float: right;
+    padding-right: 40px;
+  }
+
+  .icon2 {
+    float: right;
+    padding-bottom: 4%;
+  }
+
+  .contact {
+    background-color: #6070ff;
+    margin-top: 15%;
+    border-top-left-radius: 5pc;
+  }
+
+  .contact-me {
+    margin: 0;
+    background-repeat: no-repeat;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-size: cover;
+    text-align: center;
+    background-image: url('./images/contact\ form\ shapes-mobile.png');
+  }

```

Comment:
- [ ]  Please make sure to adhere to the `Figma` design template as much as possible as currently, your design doesn't match the `Figma` design given. This will make your design look beautiful and create a good user experience. Kindly consider the suggestions below to help improve your design:
    - [ ] Consider giving your `input` fields some more width to match that of the `Figma design`.
    - [ ] Kindly styles your **get in touch** `buttons` to match that of the `Figma` design and also remember to add a `:hover` effect using the colors provided on `Figma`.
    - [ ] Please make sure that your `contact section` is consistent across all screen sizes ( `mobile`, `table`, and `desktop`. See below for more details: 
     <table>
<tr>
<th align=“middle”>Desktop: Figma design</th>
<th align=“middle”>Desktop: Your design</th>
</tr>
<tr>
<td>
<img width="1440" alt="Screenshot 2023-05-13 at 01 19 07" src="https://github.com/hetrox8/portfolio1/assets/107192076/f41242e9-5c24-4628-aff3-23d69fa57090">
</td>
<td>
<img width="1440" alt="Screenshot 2023-05-13 at 01 18 14" src="https://github.com/hetrox8/portfolio1/assets/107192076/9d3ae0fb-a8ab-403a-b245-4c141a8622a3">
</td>
</tr>
<tr>
<th align=“middle”>Mobile: Figma design</th>
<th align=“middle”>Mobile: Your design</th>
</tr>

<tr>
<td>
<img width="375" alt="Screenshot 2023-05-13 at 01 20 02" src="https://github.com/hetrox8/portfolio1/assets/107192076/36d11052-30bc-4542-9c96-2d1ad1388c2a">
</td>
<td>
<img width="374" alt="Screenshot 2023-05-13 at 01 20 59" src="https://github.com/hetrox8/portfolio1/assets/107192076/4dd8a2cd-23ae-47e0-8e12-9cd66d82ed52">
</td>
</tr>
</table>